Python/Batch/README.md

```diff
@@ -49,6 +49,16 @@ path invocation. This sample can be run on Windows with an appropriate openssl
 binary and modified openssl invocations (i.e., `openssl.exe` instead of
 `openssl`).
 
+#### [sample4\_job\_scheduler.py](./sample4_job_scheduler.py)
+This sample shows how to run a Python script on a Windows machine. The StartTask 
```

Comment:
I think the focus for this description should be on the unique features of the Job Schedule, and how they compose with the other Batch features used: AutoPool, StartTask, Job Manager Task.

Something like this: 

This sample demonstrates how to use a Job Schedule to run recurring work. The sample creates a Job Schedule with a Job specification that has an AutoPool with a StartTask, and a JobManager Task. The Job Schedule will create a Job, at which point the AutoPool for that Job is created. The AutoPool's StartTask will run on every Compute Node, downloading and installing Python. Once completed, the Job's JobManager Task will execute, running a simple Python program. The Job will complete once all all tasks under it (here, only the Job Manager) have completed, at which point the Job Schedule is able to create the next Job recurrnence based on it's schedule.

This Job Schedule is configured to run on a recurrence, every 10 minutes, for 30 minutes in total. In that time span, 3 Jobs will be created underneath the Job Schedule, each creating their own CloudServices AutoPool. The AutoPool's lifetime is scoped to the Job, the Job terminates once all Tasks under it are completed, and the Job Schedule will terminate after the 30 minute duration.